image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -125,6 +125,8 @@ spec:
           name: collector-config
           readOnly: true
       {{- end }}
+        - mountPath: /var/profiles
+          name: collector-profiles
```

Comment:
I think these two lines need to go above `{{- end }}` because they belong to the
```
      {{- if ne ._rox.collector.collectionMethod "NO_COLLECTION"}}
      - name: collector
```
block that starts on line 57.

Your manual tests worked because this will only break the rendered manifest when `NO_COLLECTION` is used.

Reply:
You are right, I've moved the mount to the right spot, sorry I missed it!